cypress/integration/1221-my_publication/354-creator_opens_my_publications/open_my_publications.js

```diff
@@ -0,0 +1,33 @@
+import { Given, When, Then, And } from 'cypress-cucumber-preprocessor/steps';
+import { USER_WITH_AUTHOR } from '../../../support/constants';
+import 'cypress-localstorage-commands';
+
+Given('the user is logged in as Creator', () => {
+  cy.loginCognito(USER_WITH_AUTHOR).then((idToken) => {
+    cy.wrap(idToken).as('idToken');
+    cy.setLocalStorage('i18nextLng', 'eng');
+    cy.setLocalStorage('previouslyLoggedIn', 'true');
+    cy.visit('/');
+  });
+});
+When('they click the button My Publications', () => {
+  cy.get('[data-testid=my-publications').click({ force: true });
+});
+Then('they see My Publications', () => {
+  cy.get('h1').contains('My publications');

```

Comment:
risky med tekst. Og enda riskyere nå som Hege har renamet alt til `registration` isteden for `publication` 😅  Så denne kommer til å faile når ny frontend rulles ut

Reply:
Ja, men vanskelig å gjøre noe med det når scenarioet sjekker på tekst. Alternativet er å introdusere flere `data-testid`. Regner med at jeg uansett må gå gjennom testene igjen når ny frontend er rullet ut.